autoload/ctrlp/bookmarks.vim

```diff
@@ -20,16 +20,18 @@ function! ctrlp#bookmarks#init() abort
         let l:line_nrs = sort(bm#all_lines(l:file), "bm#compare_lines")
         for l:line_nr in l:line_nrs
             let l:bookmark = bm#get_bookmark_by_line(l:file, l:line_nr)
-            if l:bookmark.annotation !~ '^\s*$'
-              call add(l:text,l:bookmark.annotation)
-            else
-              call add(l:text,l:bookmark.content)
-            endif
+            let l:detail=printf("%s:%d | %s", pathshorten(l:file), l:line_nr,
```

Comment:
Sorry for being picky but could you please make sure you follow the same code formatting. In this case there are spaces missing around `=`